### PR TITLE
Add angular-gravatar.js

### PR DESCRIPTION
**Pull request by wallin**

### CODE REVIEW

File: files/angular-gravatar/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Sebastian Wallin"
+github = "https://github.com/wallin/angular-gravatar"
+homepage = "https://github.com/wallin/angular-gravatar"
+description = "Easy-to-use and flexible Angular.js directive for Gravatar images"
+mainfile = "build/angular-gravatar.min.js"
```
Comment (jimaek) [inline files/angular-gravatar/info.ini]: Just `angular-gravatar.min.js` here

File: files/angular-gravatar/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-gravatar",
+  "repo": "wallin/angular-gravatar",
+  "files": {
+    "include": ["build/angular-gravatar.js", "build/angular-gravatar.min.js"]
```
Comment (jimaek) [inline files/angular-gravatar/update.json]: No need for `build/angular-gravatar.js`

**Comment by jimaek** [comment]
Please rename the folder to `angular.gravatar`

**Comment by wallin** [comment]
Ok, I've made the changes now. Thanks!

File: files/angular.gravatar/update.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-gravatar",
+  "repo": "wallin/angular-gravatar",
+  "files": {
+    "include": ["angular-gravatar.js", "angular-gravatar.min.js"]
```
Comment (jimaek) [inline files/angular.gravatar/update.json]: You had to remove `build/` only from the info.ini. Here it needs to be present.
Remove `angular-gravatar.js` and leave only `build/angular-gravatar.min.js`

**Comment by wallin** [comment]
Ah. Got it! Sorry about that

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**